core/templates/app.constants.ts

```diff
@@ -108,8 +108,13 @@ export class AppConstants {
   // displayed.
   public static FATAL_ERROR_CODES = [400, 401, 404, 500];
 
-  // Do not modify these, for backwards-compatibility reasons.
+  // Do not modify these, for backwards-compatibility reasons. These strings are
+  // used to identify components, to generate content ids, and to determine what
+  // type of content a given content id is associated with. If you wush to
+  // change one of these, a state migration of all existing content ids is
+  // required.
   public static COMPONENT_NAME_CONTENT = 'content';
+  public static COMPONENT_NAME_INTERACTION_CUSTOMIZATION_ARGS = 'ca';
```

Comment:
Move it after HINT -- alphabetical order.

Reply:
Oops - it seems it wasn't originally alphabetical, but I've rearranged the constants here. Done